beesdoo_product/models/beesdoo_product.py

```diff
@@ -1,7 +1,7 @@
  # -*- coding: utf-8 -*-
-from openerp import models, fields, api
-from openerp.tools.translate import _
-from openerp.exceptions import UserError, ValidationError
+from odoo import models, fields, api
+from odoo.tools.translate import _
+from odoo.exceptions import UserError, ValidationError
 import uuid
 
```

Comment:
Please fix  import ordering. `uuid` is a standard python library and there fore should be imported first. See OCA guidelines : https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#imports

Also you can put a empty line between different source of import, like in the guidelines. :)